core/src/main/java/io/paradoxical/cassieq/dataAccess/QueueRepositoryImpl.java

```diff
@@ -333,19 +346,35 @@ public void deleteCompletionJob(final DeletionJob job) {
 
         if (session.execute(delete).wasApplied()) {
             logger.with(job).success("Removed deletion job");
+
+            if (deleteIfInActive(job.getQueueName())) {
+                logger.with("queue-name", job.getQueueName())
+                      .with("version", job.getVersion())
+                      .info("Deleted queue definition since it was inactive");
+            }
         }
     }
 
-
     @Override
-    public void deleteQueueStats(final QueueId id) {
+    public void deleteQueueStats(QueueSizeCounterId counterId) {
```

Comment:
I thought about it, but its not really a stats table. You can't add any other data to the table, since counter tables are hyper specific